ui/src/app/views/workspace/component.html

```diff
@@ -5,7 +5,7 @@ <h2 class="greeting">{{greeting}}</h2>
       <div role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right">
         <span *ngIf="awaitingReview" class="tooltip-content">Workspace is awaiting review</span>
         <span *ngIf="!writePermission" class="tooltip-content">Write permission required to create cohorts</span>
-        <div class="card add-card" [ngClass]="{'disabled' : !writePermission}" (click)="!writePermission || awaitingReview || buildCohort()">
+        <div class="card add-card" [ngClass]="{'disabled' : !writePermission || awaitingReview}" (click)="!writePermission || awaitingReview || buildCohort()">
```

Comment:
Looks like it's also an issue here: https://github.com/all-of-us/workbench/blob/1d33416d7eea1ecbddb6f0b7b0ae967b4d374bbf/ui/src/app/views/workspace-nav-bar/component.html#L15

Reply:
Ah, yep - you're right.